content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,634 @@
+refuse the request.
+
+My time off
+===========
+
+The :guilabel:`My Time Off` section of the *Time Off* application contains the time off dashboard,
+as well as the user's time off requests and allocations.
+
+.. _time_off/dashboard:
+
+Dashboard
+---------
+
+All users have access to the time off dashboard, which is the default view in the *Time Off*
+application. The dashboard can also be accessed at any point in the application by navigating to
+:menuselection:`Time Off application --> My Time Off --> Dashboard`.
+
+The current year is displayed, and the current day is highlighted in red. To change the view, click
+on the desired button at the top. Options are :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`,
+or :guilabel:`Year` (the default). To change the presented dates, click the left and right arrows on
+either side of the :guilabel:`Today` button. The calendar view adjusts in increments of the
+presented view. For example, if :guilabel:`Week` is selected, the arrows adjust the view by one
+week. To change the view at any point to a view that includes the current date, click the
+:guilabel:`Today` button.
+
+Above the calendar view is a summary of the users time off balances. Every time off type that has
+been allocated appears in its own summary box. Each summary lists the type of time off, the
+corresponding icon, the current available balance (in hours or days), and an expiration date (if
+applicable). To view more details on the specific time off, click the :guilabel:`? (question mark)`
+after the line `Hours (or Days) Available`. A pop up appears listing the total time allocated, the
+approved time off, the planned time off, and the total ending balance with those numbers factored
+in. The time off balance labeled as :guilabel:`Left` is what is displayed next to the icon in the
+summary. Click the :guilabel:`? (question mark)` again to close the pop-up.
+
+The legend on the right side of the calendar view displays the various time off types, with their
+corresponding colors. The status of the time off requests are shown as well. Time off that has been
+validated appears in a solid color (in the color specified in the Time Off Types part of the
+legend). Time off requests that still needs to be approved appear with white stripes in the color.
+Refused time off requests have a colored line through the dates.
+
+New time off requests can be made from the dashboard. Click the :guilabel:`New Time Off` button at
+the top of the dashboard, and a new :ref:`time off form <time_off/request-time-off>` appears. New
+allocation requests can also be made from the dashboard. Click the :guilabel:`Allocation Request`
+button at the top of the dashboard to request more time off, and a new :ref:`allocation form
+<time_off/request-allocation>` appears form appears.
+
+.. image:: time_off/dashboard.png
+   :align: center
+   :alt: Time off dashboard view with the legend, time off summaries, and view buttons highlighted.
+
+My time off
```

Comment:
```suggestion
My Time Off
```
same reason for the capitalization as before. and, once again, feel free to take or leave, I was just trying to match the rest of your doc. whichever route you choose, just be sure to keep it uniform throughout. Thanks! 👍